lib/src/react16_suggestors/react_style_maps_updater.dart

```diff
@@ -238,17 +235,16 @@ String getString({
 }
 
 String cleanString(dynamic elementToClean) {
-  return elementToClean.toString().replaceAll("\'", "").replaceAll("\"", "");
-}
+  if (elementToClean is SimpleStringLiteral) return elementToClean.value;
 
-bool nodeIsLikelyAnExpression(String node) =>
-    node.contains('?') && node.contains(':') || node.contains('??');
+  if (elementToClean is String) return elementToClean;
 
-bool isANumber(String node) => num.tryParse(node) != null;
+  return elementToClean.toString();
+}
 
-bool isAString(String node) => node.contains('"') || node.contains('\'');
+bool isANumber(String node) => num.tryParse(node) != null;
 
-bool nodeIsLikelyAVariable(String node) => !isAString(node) && !isANumber(node);
+bool isAString(dynamic node) => node is SimpleStringLiteral;
```

Comment:
Is it possible to type this as `AstNode node`, to prevent `String` values from being passed in accidentally?

Reply:
Yes sir!